### ISSUE TITLE
Missing from vscodium marketplace

### ISSUE THREAD

**Issue by edwardgalligan** [jeff-hykin/better-go-syntax#42]
I've noticed that most of the "jeff hykin" VSCode extensions have made their way over to [the VSCodium marketplace,](https://open-vsx.org/namespace/jeff-hykin) but neither `better-go-syntax` nor `better-syntax` megapack appear to be there.

I'm not sure how publishing of the others has happened (via @jeff-hykin or someone else?) so thought I'd ask here.

**Comment by jeff-hykin** [comment]
Yeah no problem. I've got an extension template, and added VSCodium support to the template a while back. I just haven't updated the template for a handful of extensions, so I'll go do that when I get a chance. Thanks for pointing it out! I also need to update the better syntax megapack anyways
